### ISSUE TITLE
Add debugging

### ISSUE THREAD

**Issue by CodeProKid** [dfmedia/DFM-Transients#12]
Should consider adding debugging throughout the codebase for when things go wrong. One option is to use the AI Debugger, but that will not be available to everyone using this plugin. Will have to think about how to approach this. Overall it would just be nice to know when / why something has gone wrong when updating a transient.

**Comment by CodeProKid** [comment]
Thinking about this some more, I think it might make the most sense to just add some hooks & filters into this codebase, and then create another plugin to do some debugging and logging.

**Comment by jasonbahl** [comment]
I agree. As some folks might want logging stored in WP and some may want the logging to be sent elsewhere
